mne_bids/sidecar_updates.py

```diff
@@ -0,0 +1,75 @@
+"""Update BIDS directory structures and sidecar files meta data."""
+# Authors: Adam Li <adam2392@gmail.com>
+#          Austin Hurst <mynameisaustinhurst@gmail.com>
+# License: BSD (3-clause)
+
+import json
+from collections import OrderedDict
+
+from mne_bids.utils import _write_json
+
+
+# TODO: add support for tsv files
+def update_sidecars(bids_path, sidecar_template_fpath, verbose=True):
+    """Update sidecar files using template JSON file.
```

Comment:
I guess something that's not yet clear from documentation is that `bids_path` can match multiple paths. It should be clarified in the `BIDSPath` docstring and also here

Reply:
Sorry I'm not following. Isn't `BIDSPath.match()` intended to return multiple paths (i.e. a list of multiple paths that match the entities)?

Reply:
> Isn't BIDSPath.match() intended to return multiple paths

right, so an analogy is that `BIDSPath` is like a regular expression/wildcard not a `str`. It returns you the list of `str` when you use `match` (similar to `re.match`). The name makes `BIDSPath` makes you think it's a single file/directory and the docstring also says "Create a BIDS path." (note the use of singular). It's not clear up front that it could match more than one path. I'm not proposing to change the name, just that it should be clarified in docstrings

Reply:
Ah I see, yes that makes sense to do!